flow/TDMetric.actor.h

```diff
@@ -317,7 +317,14 @@ auto tuple_map(F f, const Tuple &t, const Tuples &... ts) -> decltype( tuple_map
 }
 
 template <class T>
-struct Descriptor {};
+struct Descriptor {
```

Comment:
I'm failing to understand why this change is needed?  Is there something I'm unaware of about the actor compiler and metrics?

Reply:
The actor compiler generates a specialization of `Descriptor` for each actor it compiles. Therefore, this mock is needed or the code won't compile.